leasing/serializers/rent.py

```diff
@@ -47,6 +49,26 @@ class Meta:
         fields = "__all__"
 
 
+class IndexNumberYearlySerializer(serializers.Serializer):
+    id = serializers.IntegerField(required=False)
+    number = serializers.DecimalField(max_digits=8, decimal_places=1)
```

Comment:
Thanks for bringing this up. I thought about this when developing the model, and settled for just `number`, because that name was used in other index models, e.g. `leasing_index`.

Now that I google it, stat.fi offers the concept of `point figure`: https://stat.fi/meta/kas/pisteluku_en.html . That would be the more accurate name for the property, and should probably be updated in the model `IndexNumberYearly` itself, to use `point_figure` instead of `number` column.

Reply:
I would suggest the following changes:
- `IndexNumberYearly` => `IndexByYear` (because there is more to these indexes than the number)
- `OldDwellingsInHousingCompaniesPriceIndex.numbers` => `OldDwellingsInHousingCompaniesPriceIndex.indexes_by_year`
- `IndexNumberYearly.number` => `IndexByYear.point_figure`

Reply:
I agree with Juho's counter-suggestion. Sounds reasonable.

Reply:
Oh, oh, one fix to my "number" suggestion: to avoid weird-feeling operations later like `point_figure.point_figure`, maybe the column in the current `IndexNumberYearly` table should be **value** instead of either `number` or `point_figure`. Then it might be easier to understand `point_figure.value` later on.

Reply:
Sounds reasonable.

Reply:
Naming resolved in a later commit